### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by ot0tot**
FAQ section was a bit out of date.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -67,16 +58,16 @@ hide:
...
 
-## <span class="custom-heading" data-id="10">Is my binding phrase a secret?</span>
+## <span class="custom-heading" data-id="9">Is my binding phrase a secret?</span>
 
 ??? faq "Is my binding phrase a secret?"
-    No, just like what channel your VTX is on is not a secret. The binding phrase is not security, it is anti-collision. If everyone kept their VTX channel a secret, the chances of you blasting someone out of the sky accidentally is pretty high. To provide the best chance of not interfering with other pilots and them not interfering with you, be sure you're not using the same dumb bind phrase as someone else. Express your style with a hilarious or saucy bind phrase.
+    No, just like what channel your VTX is on is not a secret. The binding phrase is not used for security, it is used for anti-collision. If everyone kept their VTX channel a secret, the chances of you blasting someone out of the sky accidentally is pretty high. To provide the best chance of not interfering with other pilots and them not interferingwith you, be sure you're not using the same dumb bind phrase as someone else. Express your style with a hilarious or saucy bind phrase.
```
Comment (deadbytefpv) [inline docs/faq.md]: The binding phrase is not used for security. It is for anti-collision.

interferingwith - space needed

File: docs/faq.md [hunk 2]
```diff
@@ -10,55 +10,46 @@ hide:
...
-## <span class="custom-heading" data-id="2">Is it better than the commercial solutions?</span>
+    ExpressLRS is a high-performance, open-source radio control protocol built from the ground up to be an extremely fast (**E**xpress), extremely sensitive (**L**ong **R**ange) **S**ystem. ELRS uses Semtech SX12xx LoRa transceivers and a lightweight, highly optimized over-the-air (OTA) protocol to provide superior performance compared to legacy RC links. While ExpressLRS started with DIY and repurposed hardware, a large number of manufacturers are now producing high-quality, low-cost transmitters and receviers to make it easy for anyone to get started!
+## <span class="custom-heading" data-id="2">How does ExpressLRS compare to other systems?</span>
 
-??? faq "Is it better than the commercial solutions?"
-    It is faster than most links out there with a latency of 6.5ms (at 200hz). At 250hz 100mW 2.4GHz it is capable of ranges 30km+.
-## <span class="custom-heading" data-id="3">Which OpenTX version works with ExpressLRS</span>
+??? faq "How does ExpressLRS compare to other systems?"
```
Comment (deadbytefpv) [inline docs/faq.md:16]: plural on systems?

**Review by deadbytefpv — CHANGES_REQUESTED** [review]
Looks good so far.. I'll have the others check as well.

File: docs/faq.md (hunk 1 above)
Comment (ot0tot) [reply docs/faq.md]: Updated.

**Review by deadbytefpv — APPROVED** [review]

File: docs/faq.md [hunk 3]
```diff
@@ -10,55 +10,46 @@ hide:
...
+## <span class="custom-heading" data-id="3">Which OpenTX/EdgeTX version do I need and why should I update?</span>
 
-??? faq "Which OpenTX version works with ExpressLRS"
-    [OpenTX 2.3.12](https://www.open-tx.org/downloads.html#Releases23-ref) or newer should work just fine. No need to use OpenTX Nightly, unless you have requirements not present in the Stable versions.
+??? faq "Which OpenTX/EdgeTX version do I need and why should I update?"
+    ExpressLRS requires [OpenTX 2.3.12](https://www.open-tx.org/downloads.html#Releases23-ref) or newer, as it depends on CRSFshot (a.k.a mixer sync) to work properly. However, as OpenTX is no longer being maintained, it is strongly recommended that you update to [EdgeTX](https://github.com/EdgeTX/edgetx) in order to take advantage of the latest performance improvements and bug fixes. 
+	
+	Click [here](https://www.expresslrs.org/quick-start/transmitters/tx-prep.md) to read more about radio firmware requirements and setup.
```
Comment (jurgelenas) [inline docs/faq.md:23]: should it be relative URL instead of the absolute one?

File: docs/faq.md [hunk 4]
```diff
@@ -10,55 +10,46 @@ hide:
...
 
-    [EdgeTX 2.4.0](https://github.com/EdgeTX/edgetx) or newer should work too.
-## <span class="custom-heading" data-id="4">Why do I need to update OpenTX?</span>
+## <span class="custom-heading" data-id="4">How do I flash/update my receiver/module?</span>
 
-??? faq "Why do I need to update OpenTX?"
-    ExpressLRS needs your radio to support crsfshot (a.k.a. Mixersync) to work properly. This will give you the lowest possible latency and optimal consistency of the RC link. When your radio does not have crsfshot working, this often shows in your ExpressLRS Lua script. The Lua script top bar will show inconsistent numbers like 0:63 or is stuck at 0:250 at every packet rate rate you select.
+??? faq "How do I flash/update my receiver/module?"
+    See [Getting Started](https://www.expresslrs.org/quick-start/getting-started.md) for a Quick Setup guide and detailed instructions on flashing, features, hardware, and troubleshooting.
```
Comment (jurgelenas) [inline docs/faq.md:28]: should it be relative URL instead of the absolute one?

**Review by jurgelenas — CHANGES_REQUESTED** [review]

File: docs/faq.md (hunk 3 above)
Comment (ot0tot) [reply docs/faq.md]: Changed

File: docs/faq.md (hunk 4 above)
Comment (ot0tot) [reply docs/faq.md]: Changed

**Review by pkendall64 — APPROVED** [review]
Looks good to me

**Comment by ot0tot** [comment]
It says one change requested, but I don't see any? What needs to be done to get this merged?

**Review by maybenikhil — APPROVED** [review]

**Pull request merged**